iiod/iiod.c

```diff
@@ -296,7 +296,7 @@ static int start_iiod(const char *uri, const char *ffs_mountpoint,
 
 	IIO_INFO("Starting IIO Daemon version %u.%u.%s\n",
 		 LIBIIO_VERSION_MAJOR, LIBIIO_VERSION_MINOR,
-		 LIBIIO_VERSION_GIT);
+		 LIBIIO_VERSION_PATCH);
```

Comment:
I'm not sure this is a great idea - since it will not be set to "0" all the time - and users reporting errors will no longer know what version of iiod they are using - something from main, or a release. with the git version - everyone knew.

Reply:
For development propose and using it in other CI pipelines (like libad9361/9166, iio-osc etc), it's ok to keep using git_sha as patch version. Those daily-created packages are not used so often by customers.
For released versions, we are not allowed to use it anymore - released packages are added now in ADI Linux Package Repository and they need to have a number as patch version.

Reply:
> I'm not sure this is a great idea - since it will not be set to "0" all the time - and users reporting errors will no longer know what version of iiod they are using - something from main, or a release. with the git version - everyone knew.

If the build will be from main the git sha will appear as expected.
![version](https://github.com/user-attachments/assets/e6cd70e2-72bf-4fec-96dd-7c0a4490eb53)

Reply:
Then this is even worse - since sometimes a variable called "patch" is a githash, and sometimes it's not.